app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,34 @@
+
+<div class="form">
+<div>
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+
+</div>
```

Comment:
[ ] @Htetaungkyaw71 and @Suzi216 great work on your application however the design of your signin page doesn't match the design given kindly check the comparison below for a better understanding

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/225364583-a2ceee90-5a9f-44b6-ae78-18694cc777f9.png)  | ![image](https://user-images.githubusercontent.com/61408860/225364932-40a854c6-fe64-4c72-b341-f9c0fe7b452c.png)